src/util/numeric.h

```diff
@@ -24,42 +24,21 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 #include "../irr_v2d.h"
 #include "../irr_v3d.h"
 #include "../irr_aabb3d.h"
-#include "../threading/mutex.h"
-#include <list>
-#include <map>
-#include <vector>
-#include <algorithm>
 
+#define ARRLEN(a) (sizeof(a) / sizeof((a)[0]))
+#define MYMIN(a,b) ((a)<(b)?(a):(b))
+#define MYMAX(a,b) ((a)>(b)?(a):(b))
+#define rangelim(d, min, max) ((d) < (min) ? (min) : ((d)>(max)?(max):(d)))
+#define myfloor(x) ((x) > 0.0 ? (int)(x) : (int)(x) - 1)
+// Requires <algorithm>
+#define CONTAINS(c, v) (std::find((c).begin(), (c).end(), (v)) != (c).end())
+// The naive swap performs better than the xor version
+#define SWAP(t, x, y) do { \
```

Comment:
Yes it is impossible, take what width tabs show for you: https://forum.minetest.net/viewtopic.php?f=3&t=12247